frontend/src/components/ManageUser.tsx

```diff
@@ -0,0 +1,48 @@
+import { Col, Row } from 'react-bootstrap';
+import { useState } from 'react';
+import UserTableRow from '@components/UserTableRow';
+import datas from '@pages/user/admin/UserData.json';
+import Pagination from '@components/Pagination';
+import UserTableHeader from './UserTableHeader';
+
+const ManageUser = () => {
+  //resize
+  const [winSize, setWinSize] = useState(window.innerWidth);
+  window.addEventListener('resize', () => {
+    setWinSize(window.innerWidth);
+  });
```

Comment:
I feel like this could be solved using [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries) but I'm not sure

Reply:
i fix that by making 2 component and wrapping them in ``disapear_phone``, ``disapear_tablet`` and ``disapear_desktop``.
should i separate them into different file ?